api/enterprise-api-ref/shared-groups.markdown

```diff
@@ -177,16 +187,37 @@ curl -k --user <username>:<password> \
               "operator":"value2"
           }
       },
-      "hostsFilter": {
-          "includes": {
-              "type": "hostkey",
-              "data": ["SHA=first1"]
-          },
-          "excludes": {
-              "type": "hostkey",
-              "data": ["SHA=first2"]
-          }
+    "hostFilter": {
+      "includes": {
+        "includeAdditionally":  false,
```

Comment:
there used to be some examples of includeAdditionally true, can we add back a few examples of that to make this flag clear? Try to keep the example as small as possible right? to focus on the difference. Maybe a section all it's own for this flag since the flag is used in many places.

Reply:
I added an example of includeAdditionally to the inventory API examples, as here you cannot see any effect of having false or true.